### PR TITLE
[getting-started][docs] Fix main config in private envinronment

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

1. Updated the config for the private environment in GS.
Added parameters:
```
global: modules.https.certManager.clusterIssuerName: selfsigned
cert-manager: disableLetsencrypt
contol-plane-manager: controlPlaneConfigurator.dexCAMode: FromIngressSecret
```
2. Removed the creation of a separate Storage Class for Prometheus.
3. Added a global Storage Class for system components.
4. Fixed the text at the 3rd step of installation in a private environment.

## Why do we need it, and what problem does it solve?

This will help to improve the installation of the platform in a private environment.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Make the Getting Started more consistent.
impact_level: low
```

### CODE REVIEW

File: docs/site/_includes/getting_started/bm-private/partials/config.yml.ce.inc [hunk 1]
```diff
@@ -36,6 +36,30 @@ deckhouse:
...
         # [<ru>] Например, Grafana для %s.example.com будет доступна на домене grafana.example.com.
         publicDomainTemplate: "%s.example.com"
+        # [<en>] The HTTPS implementation used by the Deckhouse modules.
+        # [<ru>] Способ реализации протокола HTTPS, используемый модулями Deckhouse.
+        https:
+          certManager:
+            # [<en>] Name of a ClusterIssuer to use for Deckhouse modules.
+            # [<en>] For a closed environment, self-signed certificates are set.
+            # [<ru>] Имя ClusterIssuer, используемого модулями Deckhouse.
+            # [<ru>] Для закрытого окружения задаются самоподписанные сертификаты.
```
Comment (z9r5) [inline docs/site/_includes/getting_started/bm-private/partials/config.yml.ce.inc:46]:
```suggestion
            # [<en>] Use self-signed certificates for Deckhouse modules.
            # [<ru>] Использовать самоподписанные сертификаты для модулей Deckhouse.
```

File: docs/site/_includes/getting_started/bm-private/partials/config.yml.ce.inc [hunk 2]
```diff
@@ -36,6 +36,30 @@ deckhouse:
...
+            clusterIssuerName: selfsigned
+    certManager:
+      # [<en>] Disable letsencrypt and letsencrypt-staging ClusterIssuer objects (if set to true).
+      # [<en>] It is necessary to prevent attempts to access outside to Let's Encrypt.
+      # [<ru>] Не создавать ClusterIssuer letsencrypt и letsencrypt-staging в кластере.
+      # [<ru>] Необходимо для предовтвращения попыток доступа наружу к Let's Encrypt.
```
Comment (z9r5) [inline docs/site/_includes/getting_started/bm-private/partials/config.yml.ce.inc:52]:
```suggestion
```

File: docs/site/_includes/getting_started/bm-private/partials/config.yml.ce.inc [hunk 3]
```diff
@@ -36,6 +36,30 @@ deckhouse:
...
+      disableLetsencrypt: true
+    userAuthn:
+      controlPlaneConfigurator:
+        # [<en>] How to determine the CA that will be used when configuring kube-apiserver.
+        # [<en>] Extract the CA of certificate from the Secret that is used in the Ingress.
+        # [<en>] Used with self-signed certificates in Ingress resources.
+        # [<ru>] Способ определения CA, который будет использован при настройке kube-apiserver.
+        # [<ru>] Извлекает CA или сам сертификат из Secret’а, который используется в Ingress-ресурсе.
+        # [<ru>] Используется с самоподписанными сертификатами в Ingress-ресурсах.
```
Comment (z9r5) [inline docs/site/_includes/getting_started/bm-private/partials/config.yml.ce.inc:61]:
```suggestion
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Review by z9r5 — APPROVED** [review]

**Pull request merged**